fairlearn/post_processing/roc_curve_based_post_processing.py

```diff
@@ -237,7 +253,7 @@ def _roc_curve_based_post_processing_demographic_parity(attributes, labels, scor
     x_best = x_grid[i_best_DP]
 
     # create the solution as interpolation of multiple points with a separate predictor per
-    # protected attribute
+    # sensitive attribute
```

Comment:
Based on the teams thread, we might want this to be 'grouping attribute' to make it even more anodyne